vllm/compilation/backends.py

```diff
@@ -607,4 +610,9 @@ def copy_and_call(*args):
                 list_args[index] = static_tensor
             return self.split_gm(*list_args)
 
+        set_span_attributes({
```

Comment:
Can you move this higher up? There's an early return. We should just set this as soon as it's decided (line 500-510)?